docs/pages/identity-security/usage/alerts.mdx

```diff
@@ -0,0 +1,139 @@
+- Account compromises
+- Policy violations
+
+All detections are pre-configured and currently cannot be modified. The severity levels (Critical, High, Medium, Low) are defined by the Identity Security team based on the potential impact of each detection.
+
+![Alerts](../../../img/access-graph/alerts.png)
+
+
+## Detection categories
+Each category below represents an integrated service where Teleport monitors for security events and suspicious activities pulled from various integration audit logs. The detections within each category are further organized by severity level to help you prioritize your security response efforts.
+
+### AWS detections
+
+Teleport Identity Security monitors your AWS infrastructure for critical security events:
+
+#### Critical and high severity
+- Root account activities (console login, access key creation, general activity)
+- Security settings deletions (CloudTrail, GuardDuty, flow logs)
+- EBS encryption disabled
+- DB snapshots made public
+- Exposed credential policy modifications
+- Login profile updates
+
+#### Medium and low severity
+- IAM user creation
+- DB snapshot attribute modifications
+- AWS key deletion or disabling
+
+Set up AWS detections: [AWS Settings](../integrations/aws-sync.mdx)
+
+### GitHub detections
+
+Monitor your GitHub organizations and repositories for security-relevant changes:
+
+#### Critical and high severity
+- Organization security updates (SAML, MFA, OAuth restrictions)
+- Protected branch policy overrides or deletions
+- Repository access and visibility changes
+- Secret scanning alerts
+- Organization member updates
+
+#### Medium and low severity
+- Repository member updates
+- Organization moderator additions
+- Integration installations
+
+#### Advanced Security Detections
+The system includes 26 sub-types of GitHub advanced security change detections that monitor for modifications to security features like Dependabot alerts.
+
+Set up GitHub detections: [GitHub Settings](../integrations/github.mdx)
+
+### Okta detections
+
+Track identity and access management events in your Okta environment:
+
+#### High severity
+- Admin MFA disabled
+- All MFA factors reset
+- OAuth token reuse patterns
+- Rate limit violations
+- Sign-on policy evaluations
+- IDP lifecycle changes
+- API token creation/revocation
+- Security threat events
+
+#### Medium severity
+- Excessive MFA failures
+- Multiple password reset attempts
+- Dormant account access
+- Support-initiated password or MFA resets
+
+### Teleport detections
+
+Monitor Teleport-specific security events:
+
+#### High severity
+- Root SSH session initiation
+- Authentication without MFA for local accounts
+- Unusual authentication failure patterns
+- Role creation/updates/deletions
+- Connector updates
+- Unusual session commands
+
+Set up Teleport detections: [Teleport Settings](../integrations/teleport.mdx)
+
+### Cross-platform detections
+
+#### Impossible travel detection (GitHub, Okta, Teleport)
+- Severity: High
```

Comment:
We should be consistent with how we indicate severity.

In previous sections, we had a separate section header for critical, high, medium, etc.

In this section we're listing the severity as a bullet point.

Reply:
Good feedback, fixed in a commit